REQUIRE

```diff
@@ -1 +1 @@
-julia 0.6
+julia 0.7-alpha
```

Comment:
I haven't tested, but I was assuming (based on past experience, I think) that `0.7-` would mean `0.7.0` and upward. Since that hasn't been released yet there would be a version error.

Reply:
I also asked on slack after this. The answer i got from the helpdesk channel was that while technically `julia 0.7-`  does include all pre-releases, it does not work well with Femtocleaner.